src/pages/gathering/detail/components/popUp/PopUp.tsx

```
@@ -0,0 +1,42 @@
+    <div className={styles.overlay}>
+      <div className={styles.popUpWrapper}>
+        <div className={styles.popUpHeaderTitle}>
+            <p>모집 현황 ({currentPeople}/{maxPeople}명)</p>
+            <Ic_x className={styles.popUpHeaderClose} onClick={handlePopUpClose}/>
+        </div>
+     
+        <div className={styles.popUpContent}>
+            {
+                mockUp.map((item) => (
+                    <div className={styles.popUpContentItem}>
+                        <div className={styles.popUpContentItemRanking}>{item.ranking}</div>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)